### PR TITLE
Upgrade to Elasticsearch 1.5.1

### PR DESCRIPTION
**Pull request by dtan4**
## WHY

Current Elasticsearch version (**1.0.1**) in this box is too old to test the latest application.
## WHAT
- Upgrade to Elasticsearch 1.5.1 
- Drop Chef provisioning, use Docker

### CODE REVIEW

**Comment by dtan4** [comment]
@koudaiii Please review this :bow: 

I wrote some tests for normal Docker container. However, in fact, I haven't test yet how it goes as wercker box. We have to release this and test in the real wercker build process...

File: script/test-version [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+cd ${BASE_DIRECTORY}
+script/build
+
+echo "==> Running test..."
+docker run \
+  --rm \
+  --name kuromoji-elasticsearch \
+  quay.io/wantedly/kuromoji-elasticsearch:latest \
+  -v | \
+  grep -e $ELASTICSEARCH_VERSION
```
Comment (koudaiii) [inline script/test-version]: Why not use the infratester? :eyes:

Comment (dtan4) [inline script/test-version]: I found `http://target:9200/` returns version number :bulb: 

``` json
{
  "status" : 200,
  "name" : "Meanstreak",
  "cluster_name" : "elasticsearch",
  "version" : {
    "number" : "1.5.1",
    "build_hash" : "5e38401bc4e4388537a615569ac60925788e1cf4",
    "build_timestamp" : "2015-04-09T13:41:35Z",
    "build_snapshot" : false,
    "lucene_version" : "4.10.4"
  },
  "tagline" : "You Know, for Search"
}
```

**Comment by koudaiii** [comment]
LGTM!!!! :shipit:

**Comment by dtan4** [comment]
Failed to deploy...

```
Removing working directories.
Creating tar.
Package failed: Error: Command failed: tar: /var/lib/lxc/build9b3a9289-9ecf-474c-9a3a-f7223772b447: Cannot chdir: No such file or directory
tar: Error is not recoverable: exiting now
```

https://github.com/wercker/support/issues/22

> Unfortunately, the wercker-labs/docker environment can't be inherited from. It is not the same type of container as we're using in the classic environment, this means it can't be extended in the same way.

**Pull request merged**